### PR TITLE
Patch 4

### PR DESCRIPTION
**Pull request by wiese**
For the moment I only applied changes keeping original functionality, but looking forward, the token analysis also gives us a head start building a more robust way to get e.g. the data providers, maybe combined w/ 3rd party annotation parsing (doctrine/annotations).

Cheers

### CODE REVIEW

File: src/TestCounter.php [hunk 1]
```diff
@@ -30,12 +32,20 @@ public function countTotalTestsInFiles(array $files) : int
...
-
         $numTestsInFile = 0;
 
+        $stream = new PHP_Token_Stream($file);
+        $classes = $stream->getClasses();
+
+        if (!count($classes))
+        {
+            return $numTestsInFile;
+        }
```
Comment (jwage) [inline src/TestCounter.php]: Is `$classes` an array? If so, change this block of code to the following:

    if (!$classes) {
        return $numTestsInFile;
    }

If it is not an array, keep the count and move the opening { up to the same line as the if.

**Comment by jwage** [comment]
Can you test [jwage/phpchunkit-demo] against this branch? https://github.com/jwage/phpchunkit-demo

**Comment by jwage** [comment]
This will fix #9.

**Comment by jwage** [comment]
:+1:

**Comment by wiese** [comment]
```
[wiese@localhost phpchunkit-demo]$ composer.phar info jwage/phpchunkit
name     : jwage/phpchunkit
descrip. : PHPUnit Test Runner
keywords : phpunit, tests
versions : * dev-patch-4
...
[wiese@localhost phpchunkit-demo]$ docker run -v $(pwd):/app -w "/app" --rm php ./vendor/bin/phpchunkit all
PHPUnit 5.5.7 by Sebastian Bergmann and contributors.

.....                                                               5 / 5 (100%)

Time: 71 ms, Memory: 4.00MB

OK (5 tests, 5 assertions)
Total Tests: 8
Number of Chunks Configured: 14
Number of Chunks Produced: 2
Tests Per Chunk: ~1
-----------

Running chunk #1

 1/4 [🌭]  25%
 4/4 [=] 100%
Running chunk #2

 1/4 [🌭]  25%
 4/4 [=] 100%

Time: 0.12 seconds, Memory: 4MB

OK (14 chunks, 8 tests)

```

Seems to work

**Comment by jwage** [comment]
:+1: anything else you want to take care of here or are you ready to merge?

**Comment by wiese** [comment]
No amendments in this matter. If you feel it's good, please do the honours.

**Pull request merged**